editor.planx.uk/.storybook/main.js

```diff
@@ -1,25 +1,35 @@
 const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
-
 module.exports = {
   stories: ["../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  addons: ["@storybook/addon-links", "@storybook/addon-essentials"],
-  webpackFinal: async (config) => {
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/preset-create-react-app"],
+  webpackFinal: async config => {
     config.resolve.plugins ||= [];
     config.resolve.plugins.push(new TsconfigPathsPlugin());
-    config.resolve.alias["react-navi"] = require.resolve(
-      "./__mocks__/react-navi.js"
-    );
+    config.resolve.alias["react-navi"] = require.resolve("./__mocks__/react-navi.js");
     return config;
   },
-  babel: async (options) => ({
+  babel: async options => ({
```

Comment:
normally, Storybook v7 would require our project to have a `.babelrc` file, but our combo of `@storybook/react-webpack5` framework with `@storybook/preset-create-react-app` addon here fit into the "don't need to do anything" exception explained here: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#babel-mode-v7-exclusively